src/components/MoneyRequestHeader.js

```diff
@@ -187,14 +187,24 @@ const MoneyRequestHeader = (props) => {
                         titleIcon={Expensicons.Checkmark}
                         description={`${props.translate('iou.amount')} • ${props.translate('iou.cash')}${isSettled ? ` • ${props.translate('iou.settledExpensify')}` : ''}`}
                         titleStyle={styles.newKansasLarge}
+                        // Note: These options are temporarily disabled while we figure out the required API changes
+                        // shouldShowRightIcon
```

Comment:
We should update this to be `shouldShowRightIcon={!isSettled}` since we don't want to allow editing when the report is settled. We'll also need to add `disabled={isSettled}` to these.